src/io/flutter/settings/FlutterSettings.java

```diff
@@ -49,4 +50,12 @@ public boolean ignoreMismatchedDartSdks() {
   public void setIgnoreMismatchedDartSdks(boolean ignoreMismatchedDartSdks) {
     this.ignoreMismatchedDartSdks = ignoreMismatchedDartSdks;
   }
-}
\ No newline at end of file
+
+  public void setIgnoreOutOfDateSdks(boolean ignoreOutOfDateSdks) {
+    this.ignoreOutOfDateSdks = ignoreOutOfDateSdks;
+  }
+
+  public boolean ignoreOutOfDateSdks() {
```

Comment:
Convention is to use some boolean-sounding name, for example isIgnoringOutOfDateSdks()